tests/unit/test_main.py

```diff
@@ -258,7 +259,18 @@ def create_app(fetch_config, options):
         except SystemExit as exc:
             self.assertEqual(send_telemetry_ping_oop.call_count, 1)
             self.assertEqual(
-                send_telemetry_ping_oop.call_args, call("console", "firefox", telemetry_enabled)
+                send_telemetry_ping_oop.call_args,
+                call(
+                    UsageMetrics(
```

Comment:
It would be great, someday, to see you use the Glean testing API and learn what you think of that. You going through the Python bindings as a first user was a very valuable experience for us!